packages/mobile/locales/en-US/walletFlow5.json

```diff
@@ -11,7 +11,9 @@
   "wallet": "Wallet",
   "send": "Send",
   "sent": "Sent",
+  "sentTo": "Sent to",
```

Comment:
You should use the templating system and have the address param included here. We do this because not all languages will have the address follow the label.
Ex. https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/locales/en-US/sendFlow7.json#L11